client/src/lib/components/VideoLooper/VideoLooper.js

```diff
@@ -45,9 +45,12 @@ class VideoLooper extends React.Component {
   }
 
   onError(event) {
-    Sentry.captureException(event.nativeEvent, {
-      extra: {sources: this.props.sources},
-    });
+    Sentry.captureException(
+      new Error('Error in video library', {cause: event.nativeEvent}),
```

Comment:
According to Sentry `event.nativeEvent` looks like this:
```
{
  cause: "Could not load asset Error Domain=NSURLErrorDomain Code=-1100 "The requested URL was not found on this server." UserInfo={NSUnderlyingError=0x28116ba20 {Error Domain=CoreMediaErrorDomain Code=-12938 "HTTP 404: File Not Found" UserInfo={NSDescription=HTTP 404: File Not Found, NSURL=https://res.cloudinary.com/cupcake-29k/video/upload/v1679323916/Audio/Drop-Anchor-Audio-Exercise-7-minutes_tvn2wl.mp3}}, NSErrorFailingURLStringKey=https://res.cloudinary.com/cupcake-29k/video/upload/v1679323916/Audio/Drop-Anchor-Audio-Exercise-7-minutes_tvn2wl.mp3, NSErrorFailingURLKey=https://res.cloudinary.com/cupcake-29k/video/upload/v1679323916/Audio/Drop-Anchor-Audio-Exercise-7-minutes_tvn2wl.mp3, NSURL=https://res.cloudinary.com/cupcake-29k/video/upload/v1679323916/Audio/Drop-Anchor-Audio-Exercise-7-minutes_tvn2wl.mp3, NSLocalizedDescription=The requested URL was not found on this server.}, "
  target: 5793
}
```
So it should be `{cause: event.nativeEvent.cause}` right?

Reply:
Or it might be sentry that adds that... check `__serialized__` further down in this error https://29k-international-ab.sentry.io/issues/4413616365/events/34739dcfb29a4721ba79c72829648591/?project=6617535&query=is%3Aunresolved&referrer=previous-event&stream_index=11